src/app/tags/components/TagsHeader/TagsHeader.test.tsx

```diff
@@ -33,12 +128,77 @@ it("can call a function to display the add tag form", async () => {
 });
 
 it("displays the default title", () => {
-  renderWithBrowserRouter(<TagsHeader setSidePanelContent={jest.fn()} />, {
-    route: "/tags",
-    state: rootStateFactory(),
-  });
+  renderWithBrowserRouter(
+    <TagsHeader
+      filter={TagSearchFilter.All}
+      isDetails={false}
+      onDelete={jest.fn()}
+      searchText=""
+      setFilter={jest.fn()}
+      setSearchText={jest.fn()}
+      setSidePanelContent={jest.fn()}
+    />,
```

Comment:
Might be good to reduce repetition here either by using custom render function or assigning common props to a constant.